docs/proposals/multi-version.md

```diff
@@ -0,0 +1,128 @@
+# Multi-Version-Landscaper-Instances
+
+This document contains a proposal how to support multiple versions of Landscaper Instances in one Landscaper as a 
+Service (LaaS) landscape.
+
+## Motivation
+
+A LaaS landscape should support different versions of a Landscaper instance (LI), i.e. different LandscaperDeployments
+install different versions of a [this](../../.landscaper/landscaper-instance) component. A LaaS landscape should support 
+a predefined set of LI versions, which change from release to release (probably if the LaaS component version changes)
+and a customer could select when to upgrade which of his instances.
+
+A LI is a complex software artefact used by the customers for their deployments. Upgrading a LI might include complex
+things like updating the used Helm version, the kubernetes version of the Landscaper resource shoot cluster and many
+more. Due to this complexity, it is impossible to ensure that LI updates might not harm an existing customer deployments.
+
+To reduce this risk, customers must have possibility to test a new LI component version with their development setup 
+first, before they decide to upgrade the LI component of their productive environment.
+
+## Requirements
+
+This chapter defines the rules for supported LI versions in a LaaS landscape:
+
+- Versioning:
+  - Patch: A new patch version of a LI should only include bugfixes and security fixes. It should not break a customer
+    setup.
+  - Minor: A new minor version of a LI could contain additional features, minor updates of the most important components,
+    like Helm or the Garden shoot cluster. Due to its complexity a minor update might break a customer setup. 
+  - Major: Groundbreaking changes. Might break a customer setup.
+
+- Supported LI version of a LaaS landscape:
+  - The list of supported versions of a LaaS landscape only contains one patch version (usually the latest) for every 
+    supported minor version. 
```

Comment:
I think this sentence draws an incomplete picture. You are describing the supported versions later.
So we either need to add the full description on what version range is supported or remove this section and rely und the upgrade description.

Reply:
I'm not sure if I got your point but I have reworked that part with examples and some motivation. Could you check if this is ok now?

Reply:
Strange, I dont see any changes since my review.
By incomplete I meant, it doesnt tell how many minor versions/major we are supporting at the same time. However, this is described later on in the document. This made me wonder if we need this sentence at all, when there is a complete detailed description.

Reply:
Sorry now I have pushed it. I answered when I was working on the changes and push them afterwards.

Reply:
and I pressed the rereview button to early.